### DISCUSSION TITLE
Unable to login

### DISCUSSION THREAD

**Discussion by Gouri1993** [vercel/community#3267] [Help] [answered]
### Summary

Tried using login with all possible way to login, but with all way it takes me to phone number page, after entering the number it throws an error... "This phone number is already associated with a Vercel account." 

I'm resigtered thats why I'm clicking on login, but this stupid error pissing me off every time I enter my number

<img width="1425" alt="Screenshot 2023-07-17 at 10 39 39 AM" src="https://github.com/vercel/community/assets/23712732/382fcd71-670b-49b8-8f08-20715325c37f">

### Example

Screenshot attached

### Steps to Reproduce

<img width="1425" alt="Screenshot 2023-07-17 at 10 42 16 AM" src="https://github.com/vercel/community/assets/23712732/f23ffe07-2e74-40e4-9d09-418afb6793e2">

**Comment by swarnava** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username used for account creation. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
